internal/datacoord/channel_store.go

```diff
@@ -146,7 +147,11 @@ func (op *ChannelOp) BuildKV() (map[string]string, []string, error) {
 		k := buildNodeChannelKey(op.NodeID, ch.GetName())
 		switch op.Type {
 		case Add, Watch, Release:
-			info, err := proto.Marshal(ch.GetWatchInfo())
+			tmpWatchInfo := ch.GetWatchInfo()
```

Comment:
no need to Clone?

Reply:
Its very lite clone, don't want to update in place so that segmentIDs are set unexpectedly